src/plugins/intel_gna/src/transformations/remove_pre_post_processing.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/remove_pre_post_processing.hpp"
+
+#include <openvino/cc/ngraph/itt.hpp>
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/opsets/opset9.hpp>
+#include <openvino/pass/manager.hpp>
+#include <openvino/pass/pattern/op/wrap_type.hpp>
+
+#include "openvino/pass/pass.hpp"
+
+using namespace ov::opset9;
+using namespace ov::intel_gna::pass;
+
+namespace {
+
```

Comment:
Just wonder if this source file should not be part of pre_post_process subdirectory and namespace

Reply:
no, this is transformation